src/main/resources/scipooldev.properties

```diff
@@ -4,6 +4,7 @@ AWS_REGION=us-east-1
 SESSION_TIMEOUT_SECONDS=28800
 SESSION_NAME_CLAIMS=userid
 SESSION_TAG_CLAIMS=sub,userid,team,user_name,company,given_name,family_name
+REDIRECT_URIS=https://sc-dev.sageit.org/synapse,https://connect.scipooldev.org/oauth2/idpresponse
```

Comment:
Does it matter if the domain is set to `sc-dev.sageit.org` or it's source domain `https://synapse-login-scipooldev.scipooldev.org`?

Reply:
yes, it should match the redirect URI that the login app' sends to Synapse.  I believe that we use https://sc-dev.sageit.org/synapse rather than https://synapse-login-scipooldev.scipooldev.org/synapse.  Note, we could list *both* redirect URIs (and also register both redirect URIs with Synapse). Then login would work when starting at either URL.

Reply:
per issue SC-92, for "10053" ..
```
+ redirect_uris= list("https://synapse-login-scipoolprod.scipoolprod.org/synapse", "http://synapse-login-scipoolprod.scipoolprod.org/synapse"),
```

Reply:
I don't think we should be using http: .  I removed the http://... redirect URL for client 10053 (as registered in Synapse) and verified that login still works.  So we don't need to list the http:// address here or in the list registered with Synapse.